src/futuresboard/blueprint.py

```diff
@@ -321,6 +322,10 @@ def index_page():
         startdate=startdate,
         enddate=enddate,
         timeranges=ranges,
+        custom=[
```

Comment:
```suggestion
        custom=current_app.config["CUSTOM"]
```
This way you can access the values by name on the template which becomes more readable.